hack/godep-save.sh

```diff
@@ -22,40 +22,69 @@ KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
 source "${KUBE_ROOT}/hack/lib/init.sh"
 source "${KUBE_ROOT}/hack/lib/util.sh"
 
+kube::log::status "Ensuring prereqs"
 kube::util::ensure_single_dir_gopath
-kube::util::ensure_godep_version v79
 kube::util::ensure_no_staging_repos_in_gopath
 
-if [ -e "${KUBE_ROOT}/vendor" -o -e "${KUBE_ROOT}/Godeps" ]; then
-  echo "The directory vendor/ or Godeps/ exists. Remove them before running godep-save.sh" 1>&2
-  exit 1
+kube::util::ensure_godep_version
+
+TMPDIR=_tmp/godep-save.$RANDOM
```

Comment:
Okay! Thanks for your patience @thockin. Debugged this, and the issue for me is the naming of this ENVVAR. Darwin uses `TMPDIR` as a system variable. When this got to the updating bazel step, the `kube::util::go_install_from_commit` function uses the system TMPDIR and everything blows up. If you rename this to something script-specific (e.g. `TMPGODEPDIR` in my testing) then everything seems to run just fine.

Reply:
DONE!